### ISSUE TITLE
rpminspect abidiff test failing for libuv in gating, works at CLI

### ISSUE THREAD

**Issue by sgallagher** [rpminspect/rpminspect#297]
https://osci-jenkins-1.ci.fedoraproject.org/job/fedora-ci/job/rpminspect-pipeline/job/master/11459/testReport/(root)/tests/_abidiff/

```
Comparing libuv-1.41.0-1.fc35 with the older libuv-1.40.0-2.fc34 found in the "f35-updates" koji tag.

Test description:
    abidiff
When comparing two builds or two packages, compare ELF files using abidiff(1) from the libabigail project.  Differences are reported.  If the package is a rebase and not on the rebaseable list and the rebase inspection is enabled, ABI differences are reported as failures.  The assumption here is that rpminspect is comparing builds for maintenance purposes and you do not want to introduce any ABI changes for users.  If you do not care about that, turn off the abidiff inspection or add the package name to the rebaseable list.

======================================== Test Output ========================================
rpminspect version 1.3
sh: line 1:  1324 Aborted                 (core dumped) abidiff --debug-info-dir1 /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/before/x86_64/libuv-debuginfo-1.40.0-2.fc34.x86_64/usr/lib/debug/ --debug-info-dir2 /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/after/x86_64/libuv-debuginfo-1.41.0-1.fc35.x86_64/usr/lib/debug/ --headers-dir1 /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/before/x86_64/libuv-devel-1.40.0-2.fc34.x86_64/usr/include/ --headers-dir2 /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/after/x86_64/libuv-devel-1.41.0-1.fc35.x86_64/usr/include/ /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/before/x86_64/libuv-debuginfo-1.40.0-2.fc34.x86_64//usr/lib/debug/usr/lib64/libuv.so.1.0.0-1.40.0-2.fc34.x86_64.debug /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/after/x86_64/libuv-debuginfo-1.41.0-1.fc35.x86_64//usr/lib/debug/usr/lib64/libuv.so.1.0.0-1.41.0-1.fc35.x86_64.debug 2>&1
sh: line 1:  1326 Aborted                 (core dumped) abidiff --debug-info-dir1 /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/before/x86_64/libuv-debuginfo-1.40.0-2.fc34.x86_64/usr/lib/debug/ --debug-info-dir2 /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/after/x86_64/libuv-debuginfo-1.41.0-1.fc35.x86_64/usr/lib/debug/ --headers-dir1 /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/before/x86_64/libuv-devel-1.40.0-2.fc34.x86_64/usr/include/ --headers-dir2 /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/after/x86_64/libuv-devel-1.41.0-1.fc35.x86_64/usr/include/ /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/before/x86_64/libuv-1.40.0-2.fc34.x86_64//usr/lib64/libuv.so.1.0.0 /var/tmp/rpminspect/libuv-1.41.0.X1DQB1/after/x86_64/libuv-1.41.0-1.fc35.x86_64//usr/lib64/libuv.so.1.0.0 2>&1
rpminspect:
-----------
1) command line

Result: INFO
Waiver Authorization: Not Waivable

Details:
rpminspect -c /usr/share/rpminspect/fedora.yaml --arches x86_64,noarch,src --release=fc34 --tests=abidiff libuv-1.40.0-2.fc34 libuv-1.41.0-1.fc35



abidiff:
--------
1) Comparing from /usr/lib/debug/usr/lib64/libuv.so.1.0.0-1.40.0-2.fc34.x86_64.debug to /usr/lib/debug/usr/lib64/libuv.so.1.0.0-1.41.0-1.fc35.x86_64.debug in package libuv-debuginfo on x86_64 revealed ABI differences.

Result: VERIFY
Waiver Authorization: Anyone

Details:
Command: abidiff --debug-info-dir1 /before/x86_64/libuv-debuginfo-1.40.0-2.fc34.x86_64/usr/lib/debug/ --debug-info-dir2 /after/x86_64/libuv-debuginfo-1.41.0-1.fc35.x86_64/usr/lib/debug/ --headers-dir1 /before/x86_64/libuv-devel-1.40.0-2.fc34.x86_64/usr/include/ --headers-dir2 /after/x86_64/libuv-devel-1.41.0-1.fc35.x86_64/usr/include/ /before/x86_64/libuv-debuginfo-1.40.0-2.fc34.x86_64//usr/lib/debug/usr/lib64/libuv.so.1.0.0-1.40.0-2.fc34.x86_64.debug /after/x86_64/libuv-debuginfo-1.41.0-1.fc35.x86_64//usr/lib/debug/usr/lib64/libuv.so.1.0.0-1.41.0-1.fc35.x86_64.debug

abidiff: abg-dwarf-reader.cc:9364: bool abigail::dwarf_reader::compare_dies_string_attribute_value(const Dwarf_Die*, const Dwarf_Die*, unsigned int, bool&): Assertion `__abg_cond__' failed.

Suggested Remedy:
ABI changes introduced during maintenance updates can lead to problems for users.  See the abidiff(1) documentation and the distribution ABI policies to determine if this detected change is allowed.

2) Comparing old vs. new version of /usr/lib64/libuv.so.1.0.0 in package libuv on x86_64 revealed ABI differences.

Result: VERIFY
Waiver Authorization: Anyone

Details:
Command: abidiff --debug-info-dir1 /before/x86_64/libuv-debuginfo-1.40.0-2.fc34.x86_64/usr/lib/debug/ --debug-info-dir2 /after/x86_64/libuv-debuginfo-1.41.0-1.fc35.x86_64/usr/lib/debug/ --headers-dir1 /before/x86_64/libuv-devel-1.40.0-2.fc34.x86_64/usr/include/ --headers-dir2 /after/x86_64/libuv-devel-1.41.0-1.fc35.x86_64/usr/include/ /before/x86_64/libuv-1.40.0-2.fc34.x86_64//usr/lib64/libuv.so.1.0.0 /after/x86_64/libuv-1.41.0-1.fc35.x86_64//usr/lib64/libuv.so.1.0.0

abidiff: abg-dwarf-reader.cc:9364: bool abigail::dwarf_reader::compare_dies_string_attribute_value(const Dwarf_Die*, const Dwarf_Die*, unsigned int, bool&): Assertion `__abg_cond__' failed.

Suggested Remedy:
ABI changes introduced during maintenance updates can lead to problems for users.  See the abidiff(1) documentation and the distribution ABI policies to determine if this detected change is allowed.
Error: exec session exited with non-zero exit code 1: OCI runtime error
```

vs running rpminspect locally:
```
$ rpminspect -c /usr/share/rpminspect/fedora.yaml -T abidiff libuv-1.40.0-2.fc34  libuv-1.41.0-1.fc35
rpminspect:
-----------
1) command line

Result: INFO
Waiver Authorization: Not Waivable

Details:
rpminspect -c /usr/share/rpminspect/fedora.yaml -T abidiff libuv-1.40.0-2.fc34 libuv-1.41.0-1.fc35



abidiff:
--------
Result: OK
Waiver Authorization: Not Waivable
```

**Comment by dcantrell** [comment]
I believe the image needs updating.  I cannot reproduce this locally using the latest rpminspect or libabigail-1.8.1.  I'm going to try to move this issue to the rpminspect-image project, but I've never done that on github so here goes.

**Comment by dcantrell** [comment]
Nope, can only do it within my project.  @msrb what do you think about this one?

**Comment by dcantrell** [comment]
Latest release doesn't have this problem and I just tried with the development tree and it doesn't work.  I am pretty sure the CI image has been rebuilt too.  If not, open an issue at https://github.com/fedora-ci/rpminspect-image
